docs/src/plots/P3ShapeSolverPlots.jl

```diff
@@ -14,32 +14,42 @@ function guess_value(λ::FT, p1::FT, p2::FT, q1::FT, q2::FT)
 end
 
 function lambda_guess_plot(F_r::FT, ρ_r::FT) where {FT}
-    N = FT(1e8)
+    N = FT(1e6)
```

Comment:
Could you plot it for a couple of different `N`, `F_r` and `rho_r` combinations?